src/common/common-acme/src/main/java/org/niis/xroad/common/acme/AcmeProfileIdConnection.java

```diff
@@ -54,6 +54,7 @@
 @Slf4j
 public class AcmeProfileIdConnection extends DefaultConnection {
 
+    private static final String PROFILE_ID_HEADER_KEY = "profile_id";
```

Comment:
Is the profile ID header key changed (`profileID` => `profile_id`) on purpose? Has the new value been tested with the Finnish ACME test environment too?

Reply:
Yes, looks like it changed in the newer acme2certifier version, the old one without the underscore didn't work anymore, and looking at acme2certifier code seems to confirm this. Retested it with the Finnish ACME test environment and it worked with the newer "profile_id"